### ISSUE TITLE
DuplicateProperty consecutive duplicates

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#715]
In #114, after it was closed, a discussion began regarding how CSSLint [ignores consecutive duplicates](https://github.com/CSSLint/csslint/wiki/Disallow-duplicate-properties). The idea is that if you write:

```css
a {
  background: #fff;
  background: rba(255, 255, 255, .5);
}
```

then you probably did so intentionally, since its visually so easy to see that the same property is defined twice. If you hadn't meant this to happen, you would have seen it and fixed it.

I'd be happy to contribute a config for DuplicateProperty, like `ignore_consecutive` or `allow_consecutive`, or something.

**Comment by srawlins** [comment]
Oh, my specific motivation here is for defining multiple `transition` properties, as the duration and timing function gets pretty long for each property when using variables:

```scss
a {
  transition: opacity $some-speed $some-timing-function;
  transition: background-color $some-other-speed $some-other-timing-function;
}
```

**Comment by sds** [comment]
Definitely open to a pull request! 

`ignore_consecutive` seems like a clear option name to me.
